.github/workflows/image-push-master.yaml

```diff
@@ -0,0 +1,126 @@
+name: "push images on merge to master"
+
+env:
+  IMAGE_NAME: ghcr.io/${{ github.repository }}
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-and-push-amd64-rdma-cni:
+    name: image push amd64
+    runs-on: ubuntu-20.04
```

Comment:
any reason why not to use ubuntu-latest

Reply:
essentially copied either from sriov-cni or sriov-device-plugin.

maybe for stability ? so u know what u get when building image.

that said. i can change to latest if u think it better :)

Reply:
it good to pin it, what go me confused is that the build and test are on ubunut-latest. 
Maybe we should move all to ubuntu-22.04

Reply:
ack. will pin all and move to 22.04.